sdk/python/kfp/compiler_cli_tests/test_data/pipeline_with_task_final_status_yaml.json

```diff
@@ -66,7 +66,7 @@
             "pipeline status:",
             "{{$.inputs.parameters['status']}}"
           ],
-          "image": "python:3.7"
+          "image": "python:3.9"
```

Comment:
revert this change also? (including the .py and .json file)

Reply:
Done